app/src/main/java/de/grobox/transportr/networks/Region.java

```diff
@@ -56,6 +56,7 @@
 	CANADA(R.string.np_region_canada, "🇨🇦"),
 	COSTA_RICA(R.string.np_region_costa_rica, "🇨🇷"),
 	AFRICA(R.string.np_region_africa, "🌍");
```

Comment:
has to end with a comma here because the enum continues in the next line

Reply:
Yes, indeed.